### PR TITLE
Corrections and Add files

### PR DESCRIPTION
**Pull request by hellresistor**
Corrections on translation language and links mapping
Added **about.html** in **pt** and **es** folders

### CODE REVIEW

**Comment by hellresistor** [comment]
Hope its this ^^

**Comment by hellresistor** [comment]
A Issue, in `pt/index.html` file Not Showing *coin Icon.

I dont understand javascript. But detect if addig inside down line the `||'pt'` , is fixed for pt page , but messed for english mainpage, spanish page stays ok.  :X 

File: `./js/modules/calculator/index.js`
Line: `62`
Code: `let prefix = window.location.pathname.split('/').filter(s => s && s !== 'index.html').pop() === 'es'||'pt'? '..' : '.'`

File: es/pages/about.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            <li><a href="../../es/pages/best-coins-to-mine.html">Moedas para minar</a></li>
+            <li><a href="#">Sobre</a></li>
+        </ul>
+    </nav>
+
+    <main>
+        <div class="about">
+            <h2>About</h2>
+                <p class="title">Sobre</p>
+                    <p>Este é um projecto de código aberto</p>
```
Comment (Lmendev) [inline es/pages/about.html:34]: This doesn't looks like spanish 🥲

**Review by Lmendev — APPROVED** [review]
I think it's okay
I can add features in another pull request

File: es/pages/about.html (hunk 1 above)
Comment (hellresistor) [reply es/pages/about.html:34]: the friend have solved :P

**Pull request merged**